generator/src/main/java/com/intendia/openapi/Main.java

```diff
@@ -138,7 +145,21 @@ public static SpecData valueOf(String apiVersion) {
                 @Override TypeName wrap(TypeName t) { return ArrayTypeName.of(t); }
             },
             LIST {
-                @Override TypeName wrap(TypeName t) { return ParameterizedTypeName.get(ClassName.get(List.class), t); }
```

Comment:
I think that you can just add `.box()`? try it out, please 😄
```
return ParameterizedTypeName.get(ClassName.get(List.class), t.box());
```
so the switch is not needed...

Reply:
Sorry I overlooked that one... It compiles and looks like it should be fine.
I'm not using List arguments in my API definition any more so I don't have a test case. What I was trying to do I decided was actually a bad idea - the idea just came about by mirroring a legacy API.

We can drop it if you want.